packages/core/src/execute/BaseRoutine.ts

```diff
@@ -79,45 +89,25 @@ export default abstract class BaseRoutine<Ctx extends Context<BaseContextArgs>>
   /**
    * Group routines in order of which they are dependend on.
    */
-  orderByWorkspacePriorityGraph(): {
-    other: Routine<Ctx, BeemoTool>[];
-    priority: Routine<Ctx, BeemoTool>[];
-  } {
+  orderByWorkspacePriorityGraph(): Routine<Ctx, BeemoTool>[][] {
     const enabled = this.context.args.priority || this.tool.config.execute.priority;
 
     if (!enabled || !this.context.args.workspaces) {
-      return {
-        other: this.routines,
-        priority: [],
-      };
+      return [this.routines];
```

Comment:
Should probably just be `return this.routines`.

Reply:
this is intentional, if priority is disabled, we want to run them all in parallel right?  To do that, we return a single batch with all routines.  This method was updated to return an an array of arrays to match the return value of resolveBatchList

Reply:
Derp, keep forgetting these are batched.